.github/ISSUE_TEMPLATE/8_design_doc.yml

```diff
@@ -0,0 +1,32 @@
+name: Submit a Flutter design document.
+description: As a contributor, I would like to submit a design document.
+title: '[Design Document] <title>'
+labels: ['design doc']
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for your interest in contributing to Flutter!
+
+        Please ensure that you are following
+        https://github.com/flutter/flutter/wiki/Design-Documents
+        when submitting a design document.
+
+          Please fill out the sections below to the best of your ability.
+  - type: input
+    id: document_link
+    attributes:
+      label: Document Link
+      description: |
+        Insert the 'go/link' for your document here, as referenced in
+        https://flutter.dev/go/template
+    validations:
+      required: true
+  - type: textarea
+    id: proposal_description
+    attributes:
+      label: Proposal Description
+      description: |
+        Please provide a brief description of what you are proposing.
```

Comment:
For consistency with the design doc template, maybe rather than looking for the proposal description here we should ask "what problem are you solving", that might help guide people who are new to design docs.